restcord/invite.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from .user import User
+
+__all__ = (
+    'Invite'
+)
+
+class Invite:
+
+    """
+    Model depicting a Discord invite object.
+    """
+
+    __slots__ = ('code', 'guild', 'channel', 'inviter', 'target_user', 'target_user_type', 'approximate_presence_count', 'approximate_member_count')
+
+    def __init__(self, **kwargs):
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/manual/Yandawl/restcord.py/pullRequest?prid=5910681)